src/docs/command-line.dox

```diff
@@ -512,25 +512,30 @@ The \c write command stores records into the specified data source.
 The data source must be configured with string or record number keys and
 string values.
 
-If the \c write command is called with the \c -a option, each
-command-line argument is a single value to be appended to the specified
-column-store data source.  If the \c write command is not called with
-the \c -a option, the command-line arguments are key/value pairs.
+If the \c write command is called with the \c -a option, each command-line
+argument is a single value to be appended to the specified column-store
+data source. If the \c write command is called with the \c -r option,
+the command-line argument is an existing key. If the \c write command is
```

Comment:
Does it need to be an existing key with `-r` option?

Reply:
Actually one can provide any key, and only an existing key is meaningful to remove a record, a non-existing key would lead to a WT_NOTFOUND error. I can update the text by removing the "existing" word.

Reply:
Yes, please I would like that. The user is free to give any key.

Reply:
Updated in [d8c1c28](https://github.com/wiredtiger/wiredtiger/pull/8906/commits/d8c1c28361791483d2c666019b3b37a180c0e942)